pulsar-broker/src/main/java/org/apache/pulsar/broker/web/PulsarWebResource.java

```diff
@@ -1051,7 +1050,7 @@ public void validateTopicOperation(TopicName topicName, TopicOperation operation
                 throw new RestException(Status.UNAUTHORIZED, "Need to authenticate to perform the request");
             }
 
-            AuthenticationDataHttps authData = clientAuthData();
+            AuthenticationDataSource authData = clientAuthData();
```

Comment:
AuthenticationDataHttps is the return value of the error, use AuthenticationDataSource instead of it